Subject: [PATCH] SystemUI:Bluetooth can't be turn off

Bluetooth can't be turn off after doule
turn on/off Bluetooth butooth in quick
settings,when bluetooth is in state turing
on toggle will be set to unable,when you
stay in datail page icon will never refresh
again,so add the refresh logic.

diff --git a/packages/SystemUI/src/com/android/systemui/qs/tiles/BluetoothTile.java b/packages/SystemUI/src/com/android/systemui/qs/tiles/BluetoothTile.java
index f1e8749..d1ff20c 100644
--- a/packages/SystemUI/src/com/android/systemui/qs/tiles/BluetoothTile.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/tiles/BluetoothTile.java
@@ -112,6 +112,9 @@
         state.autoMirrorDrawable = false;
         state.minimalContentDescription =
                 mContext.getString(R.string.accessibility_quick_settings_bluetooth);
+        if (mController.getBluetoothState() == BluetoothAdapter.STATE_ON) {
+            fireToggleStateChanged(true);
+        }
         if (enabled) {
             state.label = null;
             if (connected) {
